docs/README.md

```diff
@@ -57,50 +57,52 @@ For additional bot examples see [`examples`](https://github.com/telegraf/telegra
-* [OCR_Tool_Bot: Tesseract recognition](https://github.com/Piterden/tesseract-bot)
-* [OneQRBot: scan and generate QR](https://github.com/Khuzha/oneqrbot)
-* [Ordis Prime](https://github.com/MaxTgr/Ordis-Prime)
-* [PodSearch bot(TypeScript)](https://fazendaaa.github.io/podsearch_bot/)
-* [RandomPassBot: generate a password](https://github.com/Khuzha/randompassbot)
-* [Randy](https://github.com/backmeupplz/randymbot)
-* [Referal system: channels promoter](https://github.com/Khuzha/refbot)
-* [scrobblerBot](https://github.com/drvirtuozov/scrobblerBot)
-* [Shieldy](https://github.com/backmeupplz/shieldy)
-* [SimpleRegBot](https://github.com/Khuzha/simpleRegBot)
-* [Spyfall Game Bot](https://github.com/verget/telegram-spy-game)
-* [Stickers Play Bot](https://github.com/TiagoDanin/StickersPlayBot)
-* [Store Of Bot](https://github.com/TiagoDanin/StoreOfBot)
-* [Syntax Highlighter Bot](https://github.com/piterden/syntax-highlighter-bot)
-* [telegraf-rutracker-transmission](https://github.com/DZamataev/telegraf-rutracker-transmission)
-* [telegram-telegraf-bot](https://github.com/Finalgalaxy/telegram-telegraf-bot)
-* [Temply](https://github.com/backmeupplz/temply)
-* [TereGramBot](https://github.com/juandjara/TereGramBot)
-* [The Guard Bot](https://github.com/TheDevs-Network/the-guard-bot)
-* [ThemerBot](https://github.com/YouTwitFace/ThemerBot)
-* [TTgram](https://github.com/TiagoDanin/TTgram)
-* [Voicy](https://github.com/backmeupplz/voicy)
-* [Watchy](https://github.com/backmeupplz/watchy)
-* [yt-search-bot](https://github.com/Finalgalaxy/yt-search-bot)
-* [YTubevideoBot](https://github.com/n1ghtw0lff/YTubevideoBot)
-* Send PR to add link to your bot
+| Name | Description |
+| --- | --- |
+| [BooksAndBot](https://github.com/dmtrbrl/BooksAndBot) |   |
+| [CaptchaOnlyBot](https://github.com/Piterden/captcha_only_bot) | Configurable question \w set of buttons on a new group user |
+| [ChannelHashBot](https://github.com/YouTwitFace/ChannelHashBot) |   |
+| [ChatLinkerBot](https://github.com/jt3k/chat-linker) | The bridge between jabber and telegram |
+| [ChessBot](https://github.com/Piterden/chessbot) | Inline chess game in a message |
+| [CounterBot](https://github.com/leodj/telegram-counter-bot) |   |
+| [DefendTheCastle](https://github.com/TiagoDanin/Defend-The-Castle) |   |
+| [EveMoviesBot](https://github.com/dmbaranov/evemovies-bot) |   |
+| [GNU/LinuxIndonesiaBot](https://github.com/bgli/bglibot-js) |   |
+| [GoogleItBot](https://github.com/Edgar-P-yan/google-it-telegram-bot) | Instant inline search |
+| [GroupsAdminBot](https://github.com/Azhant/AdminBot) |   |
+| [KitchenTimerBot](https://github.com/DZamataev/kitchen-timer-bot) |   |
+| [LyricsGramBot](https://github.com/lioialessandro/LyricsGramBot) | Song Lyrics |
+| [MangadexBot](https://github.com/ejnshtein/mangadex_bot) |   |
+| [Memcoin](https://github.com/backmeupplz/memcoin) |   |
+| [MetalArchivesBot](https://github.com/amiralies/metalarchives-telegram-bot) |   |
```

Comment:
sth like `Unofficial metal-archives.com telegram bot`

Reply:
Could you push it pls?

Reply:
To this [branch](https://github.com/Piterden/telegraf/tree/patch-3)

Reply:
access?

Reply:
Make a PR, I will merge

Reply:
Make a PR

Reply:
Nice thnx